### ISSUE TITLE
Task: Find someone external

### ISSUE THREAD

**Issue by sjinko** [CUPUM/ai-edi-space#32]

**Comment by myriamgui** [comment]
@sjinko, L'offre de service de Karen et Célia me semble valide dans son ensemble. Toutefois, plusieurs éléments tels que la revue de littérature, la validation des espaces, thématiques et groupes cibles devraient être réalisés par la Chaire. Cela permettrait ainsi de réduire le mandat des experts qui est assez large dans cette proposition (et dispendieux).

**Comment by sjinko** [comment]
Pourrais-tu leur demander de ventiler le budget et de préciser ce qu’elles entendent par revue de la littérature et validation des thématiques et groupes ciblés ?

Shin Koseki
Titulaire de la Chaire UNESCO en paysage urbain /
  UNESCO Chair Professor in Urban Landscape
Université de Montréal
________________________________

**Comment by myriamgui** [comment]
C'est expliqué dans leur offre de service : 

-Tour de la littérature portant sur les populations marginalisées en lien avec espaces publics urbains 
-Revue de la littérature selon les thématiques pertinentes (sécurité, confort, inclusivité etc.)
-Validation des marqueurs de diversité pertinents (ex: genre, âge, religion, handicap, racialisation, classe ?)

C'est ce que l'on fait déjà actuellement

**Comment by sjinko** [comment]
Oui, c’est ce que je me dis aussi. Mais j’aimerais savoir quelle portion du budget va dans chacune de ces tâches pour voir dans quelle mesure elles ont bien compris le mandat
________________________________

**Comment by myriamgui** [comment]
@sjinko, L'offre pour l'agent.e de recherche est ici :

https://cirrus.gnp.umontreal.ca/index.php/s/LwYqWXy65jRnw7K

**Comment by sjinko** [comment]
Ça me semble très bien. Est-ce qu’on l’envoie à C. Paré pour avis?
________________________________
